runtime/doc/api.txt

```diff
@@ -1350,6 +1350,57 @@ nvim_out_write({str})                                       *nvim_out_write()*
                 Parameters: ~
                     {str}  Message
 
+nvim_parse_cmd({str}, {opts})                               *nvim_parse_cmd()*
+                Parse command line.
+
+                Doesn't check the validity of command arguments.
+
+                Attributes: ~
+                    {fast}
+
+                Parameters: ~
+                    {str}   Command line string to parse. Cannot contain "\n".
+                    {opts}  Optional parameters. Reserved for future use.
+
+                Return: ~
+                    Dictionary containing command information, with these
+                    keys:
+                    • cmd: (string) Command name.
+                    • line1: (number) Starting line of command range.
+                    • line2: (number) Final line of command range.
+                    • force: (boolean) Whether command contains a bang.
+                    • args: (array) Command arguments.
+                    • addr: (string) Value of |:command-addr|. Uses short
+                      name.
+                    • nargs: (string) Value of |:command-nargs|.
+                    • nextcmd: (string) Next command if there are multiple
+                      commands separated by a |:bar|. Empty if there isn't a
+                      next command.
+                    • magic: (array) Which characters have special meaning
+                      inside the command arguments. Possible values inside the
+                      array are:
+                      • '': '' and '#' get expanded by the command.
```

Comment:
`%` is missing here. perhaps it needs escaped as `%%` in source?

Reply:
fixed